common/helpers/campCollaborationDisplayName.js

```diff
@@ -14,7 +14,8 @@ export default function (campCollaboration, tc, indicateInactive = true) {
       : campCollaboration.inviteEmail || ''
 
   if (campCollaboration.status === 'inactive' && indicateInactive) {
-    text += ' (' + tc('entity.campCollaboration.inactive') + ')'
+    const inactiveIndicator = tc ? tc('entity.campCollaboration.inactive') : 'inactive'
```

Comment:
This prevents the exception, but allows an invalid usage of this method. So it hides errors. 
Rather, I would throw an explanatory exception.

#3435  should cover all incorrect usages.

Reply:
If you can assure me that all current and future usages will use this function correctly, then i am with you. (I think you can't).
Sadly v-calendar swallows the NPE which happened here completely in production and we have no sentry event which indicates that something went wrong. This would also swallow any error i throw here. (In dev you see the error in the console).

Very important to me is the change in dev data.

I would also rather have a bad default than not appearing ScheduleEntries in prod.
I would offer to add a console.error here and test that this appears in sentry?

Reply:
How about we create two separate functions, one which renders the inactive state and requires the tc argument, and one which ignores the inactive state and does not accept a tc argument?

Reply:
> I would offer to add a console.error here and test that this appears in sentry?

I'd vote for this. Otherwise we will not detect such cases and we might generate other issues elsewhere.

This is a good example where Typescript would/will generate some value, if we ever introduce in future. It would force you to always pass the `tc` parameter.